Runtime/JSCallbackArgs.cs

```diff
@@ -1,51 +1,57 @@
+using System;
 using System.Runtime.InteropServices;
 using static NodeApi.JSNativeApi.Interop;
 
 namespace NodeApi;
 
-public class JSCallbackArgs
+public readonly ref struct JSCallbackArgs
```

Comment:
>ref struct  [](http://example.com/codeflow?start=16&length=11)

Can we use 'struct' instead of 'ref struct'?

Reply:
It has to be `ref struct` so that it can have a `Span` member, to support stack-allocation of the args array.